docs/source/notes/autograd.rst

```diff
@@ -219,24 +219,58 @@ Autograd for Complex Numbers
 **What notion of complex derivative does PyTorch use?**
 *******************************************************
 
-PyTorch follows `JAX's <https://jax.readthedocs.io/en/latest/notebooks/autodiff_cookbook.html#Complex-numbers-and-differentiation>`_
-convention for autograd for Complex Numbers.
+PyTorch uses the Wirtinger Calculus to compute gradients for functions with complex valued input and (or)
+output. Wirtinger Calculus provides a means of computing gradients of real valued cost functions defined on
+complex domains(:math:`ℂ^n`) which are not holomorphic (complex differentiable). This comes in very handy since
+most of the commonly used linear algebraic functions are not holomorphic.
```

Comment:
I am not sure, if the statement
 `... most of the commonly used linear algebraic functions are not holomorphic.`
will be correctly interpreted from a user.
Actually many complex operations are holomorphic.
The drawback is, that there is no relevant holomorphic function to end up with a cost function (i.e. fulfill the Cauchy-Riemann equations).